overlay/files/system/oem/00_datasource.yaml

```diff
@@ -5,24 +5,27 @@ stages:
       if: '[ ! -f /oem/userdata ]'
       name: "Pull data from provider"
       datasource:
-        providers: ["cdrom", "gcp", "openstack", "aws", "azure", "hetzner", "packet", "scaleway", "vultr", "digitalocean", "metaldata"]
+        providers: ["cdrom", "gcp", "openstack", "aws", "azure", "hetzner", "packet", "vultr", "digitalocean", "metaldata"]
```

Comment:
scaleway has no timeout out of the box so it can take up to 60 seconds to timeout if there is network during initramfs. Patch upstream to fix this has been sent and merged, waiting for it to trickle down.